packages/@sanity/portable-text-editor/src/editor/__tests__/PortableTextEditor.test.tsx

```diff
@@ -310,7 +310,7 @@ describe('initialization', () => {
         _key: 'abc',
         _type: 'myTestBlockType',
         markDefs: [],
-        children: [{_key: 'def', _type: 'span', text: 'Test', marks: ['invalid']}],
+        children: [{_key: 'def', _type: 'span', marks: []}],
```

Comment:
The previous test case is now auto-resolved, so we need another validation error for this test.